src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,12 @@ Release History
 
 * Support cluster certificate rotation operation using "az aks rotate-certs".
 
+**AppConfig**
+
+* Add support for using ":" for `as az appconfig kv import` separator
+* Fix issue for listing key values with multiple labels including null label. 
+* Apply latest version of management plane sdk. 
```

Comment:
Can you specify you package name and version number here?

Reply:
Sure thing.